### ISSUE TITLE
mapping to http://schema.org/SoftwareApplication

### ISSUE THREAD

**Issue by elf-pavlik** [ewilderj/doap#20]
i see mention of work on mapping foaf:Person to schema:Person
maybe we could find useful mapping of doap:Project to schema:SoftwareApplication ?

related: #9

**Comment by ewilderj** [comment]
Yes, DanBri and I briefly talked earlier this week about schema.org integration/mapping. I will set up some time to chat with him about this.

**Comment by csarven** [comment]
There is `https://dokie.li/ a doap:Project`. (Aside: It should probably be `https://dokie.li/#project a doap:Project` instead.)

I'm approaching this with an interest to use `prov:wasGeneratedBy`/`as:generator` where it points at either the "project" or "application". doap:Project seems to include/acknowledge the concept of application, but probably more since it is not only about the application itself.

**Comment by kjetilk** [comment]
Yeah, I'm not sure I feel ```doap:Project``` and ```schema:SoftwareApplication``` to be easily mapped, as @csarven says, a project is not just about the application. But then, ```owl:sameAs``` is not the only option for mapping, and I'm sure @danbri would have a clear and thoughful opinion :-)

**Comment by csarven** [comment]
I think this issue should be rephrased to potential relationships between the concepts, and not necessarily a *particular* "mapping". Understanding and clarifying the intentions and boundaries of what's here is more useful I think. Mapping may or may not be necessary.

**Comment by olberger** [comment]
I think there used to be a confusion between project and program in various uses of DOAP, but I'd like to draw attention to the fact that if one wants to model organisations developers and software, then it would be interesting to differentiate the project (as an organisation) developing the software, from the software itself.

Of course this is just rethorical, and my work in this field dates back a few years (cf. https://www-public.tem-tsp.eu/~berger_o/weblog/tag/doap/)

**Comment by danbri** [comment]
So foaf:Person and schema:Person mean essentially the same thing, the definitions are pretty close.

Last time we rev'd the FOAF spec I put in some of the more obvious mappings, although only the Person one is also declared within schema.org's data files.

Since DOAP was built alongside FOAF we probably ought to take the opportunity to do a quick compare/contrast on any things still missing from schema.org that were in FOAF. Organization is in both. We don't need a lot of the FOAF classic stuff (dnaChecksum's time may yet come, but er, no thanks; mbox_sha1sum needs mothballing etc.). 

I've never been sure what to do about Project, Group and Organization. FOAF had all 3. Currently [Project](xmlns.com/foaf/0.1/#term_Project) is not a subtype of Organization although you could make the case for that. Group is also an oddity, in that it supports a) lists of people (e.g. bookmarked or followed on Twitter or categories in a contacts manager) b) lists of people who are actually in some sense aware they are a group. 

My suggestion is that we could add "Project" to schema.org as a subtype of Organization and then figure out how projects and software applications relate.

Another topic I've been meaning to raise with @ewilderj is the line between software distributions and datasets, which is becoming more blurred over time...

**Comment by elf-pavlik** [comment]
Somehow I have hard time to see that those properties with `rdfs:domain doap:Project` describe an Organization and not SoftwareApplication (the code base not particular deployment)

```turtle
@prefix doap: <http://usefulinc.com/ns/doap#> .
@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .

doap:release
  rdfs:domain doap:Project ;
  rdfs:range doap:Version .

doap:repository
  rdfs:domain doap:Project ;
  rdfs:range doap:Repository .

doap:download-page
  rdfs:domain doap:Project .

doap:bug-database
  rdfs:domain doap:Project .

doap:screenshots
  rdfs:domain doap:Project .

doap:programming-language
  rdfs:domain doap:Project .

doap:implements
  rdfs:domain doap:Project ;
  rdfs:range doap:Specification .

doap:vendor
  rdfs:domain doap:Project .
```

**Comment by ewilderj** [comment]
Does anybody care about this still? It doesn't seem that we ever came to a conclusion of any kind.

**Comment by danbri** [comment]
Fwiw
https://schema.org/Project now exists, as anticipated in my 2017 post

**Comment by kjetilk** [comment]
> Fwiw https://schema.org/Project now exists, as anticipated in my 2017 post

Then, I think it makes sense to map this.
